scripts/release.py

```diff
@@ -13,52 +18,55 @@ def getValue(var, line):
     return val
 
 
-content = "content"
-config = f"{content}/js/config.js"
-f = open(config, "r", encoding="utf-8")
+config = f"js/config.js"
+with open(config, "r", encoding="utf-8") as f:
+    pubDomain = ""
+    shortName = ""
+    publishVersion = ""
 
-pubDomain = ""
-shortName = ""
-publishVersion = ""
+    for line in f:
+        if "pubDomain" in line:
+            val = getValue("pubDomain", line)
+            if val:
+                pubDomain = val
+        elif "shortName" in line:
+            val = getValue("shortName", line)
+            if val:
+                shortName = val
+        elif "publishVersion" in line:
+            val = getValue("publishVersion", line)
+            if val:
+                publishVersion = val
 
-for line in f:
-    if "pubDomain" in line:
-        val = getValue("pubDomain", line)
-        if val:
-            pubDomain = val
-    elif "shortName" in line:
-        val = getValue("shortName", line)
-        if val:
-            shortName = val
-    elif "publishVersion" in line:
-        val = getValue("publishVersion", line)
-        if val:
-            publishVersion = val
-f.close()
-os.remove(config)
+SPECIFICATION_FOLDER = CURRENT_WORKING_DIRECTORY / "publicatie" / pubDomain / shortName
 
-if len(pubDomain) * len(shortName) > 0:
+def create_redirect_for_version(publishVersion):
+    with open(f'{SPECIFICATION_FOLDER}/index.html', 'w') as root_index_file:
+        root_index_file.write(f"""<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Redirecting to v{publishVersion}</title>
+<meta http-equiv="refresh" content="0; URL=./{publishVersion}">
+<link rel="canonical" href="./{publishVersion}">
+""")
+
+    version_without_last_number = ".".join(re.split("\.", publishVersion)[:-1])
+
+    with open(f'{SPECIFICATION_FOLDER}/{version_without_last_number}/index.html', 'w') as version_redirect_file:
+        version_redirect_file.write(f"""<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Redirecting to v{publishVersion}</title>
+<meta http-equiv="refresh" content="0; URL=../{publishVersion}">
+<link rel="canonical" href="../{publishVersion}">
+""")
+
+
+if len(pubDomain) * len(shortName) * len(publishVersion) > 0:
     try:
-        if len(publishVersion) > 0:
-            shutil.copytree(content, f"{content}/{publishVersion}")
-        path = f"publicatie/{pubDomain}/{shortName}/"
-        if not os.path.exists(path):
-            os.makedirs(path)
-        for fn in os.listdir(path):
-            # https://stackoverflow.com/a/185941
-            file_path = os.path.join(path, fn)
-            try:
-                print("file_path: " + file_path)
-                if os.path.isdir(file_path):
-                    files = os.listdir(file_path)
-                    if "index.html" in files and fn != publishVersion:
-                        shutil.copytree(file_path, f"{content}/{fn}")
```

Comment:
Eerst kopieerden we alle bestanden meerdere keren, omdat we het path "leeg maakten". Nu laten we de huidige bestanden intact en verwijderen we ze enkel als het pad al bestond.